File: sdkwork-backend-sdk-go/types/plus_account_vo.go
package types

// 用户账户VO，记录余额和积分信息
type PlusAccountVO struct {
	CreatedAt string `json:"createdAt"`
	UpdatedAt string `json:"updatedAt"`
	UserId int `json:"userId"`
	AvailableBalance float64 `json:"availableBalance"`
	FrozenBalance float64 `json:"frozenBalance"`
	AvailablePoints int `json:"availablePoints"`
	FrozenPoints int `json:"frozenPoints"`
	Status string `json:"status"`
	HistoryList []PlusAccountHistoryVO `json:"historyList"`
}

File: sdkwork-backend-sdk-python/sdkwork_backend_sdk/models/plus_account_vo.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class PlusAccountVO:
    """用户账户VO，记录余额和积分信息"""
    created_at: str = None
    updated_at: str = None
    user_id: int = None
    available_balance: float = None
    frozen_balance: float = None
    available_points: int = None
    frozen_points: int = None
    status: str = None
    history_list: List[PlusAccountHistoryVO] = None
